part-time-schedule/week-1.md

```diff
@@ -0,0 +1,87 @@
+---
+description: Building on bitcoin and setting up a dev environment
+---
+
+# Week 1
+
+## Prerequisites
+
+We would like you to download some source code and attempt to compile it by following its build instructions before the course begins. This is to prevent any delays when starting on the day.
+
+Please download/clone the following tools/repositories and try to follow and complete their build processes:
+
+| Software       | Repository                                                                                   |
+| -------------- | -------------------------------------------------------------------------------------------- |
+| NodeJS         | [https://nodejs.org/en/download/](https://nodejs.org/en/download/)                           |
```

Comment:
I'd be okay with leaving this out?

Reply:
Given how familiar with javascript and typescript people seem to be, I'd say yes